Source/Tests/LinuxSyscalls/x64/Syscalls.cpp

```diff
@@ -153,6 +156,8 @@ namespace FEX::HLE::x64 {
 #endif
     }
 
+    syscalls_x64.clear();
```

Comment:
Haven't had a chance to look into this in detail, but mind you `.clear()` doesn't free the vector memory. You'll want something like `syscalls_x64 = {}` for that instead.

Reply:
...What a fucking nightmare footgun that is.

Reply:
Resolved now though.

Reply:
Seems like only the change made it through for the syscalls_x32 vector. Was that intended?

Reply:
Oops, yep, not intentional. Fixed now.